docs/artefacts/offramps.md

```diff
@@ -484,6 +485,57 @@ Compare And Swap operation. Those operations require old values to match what it
 }}
 ```
 
+### nats
+The nats offramp connects to nats server(s) and publishes a message to specified subject for every event.
+
+The default [codec](codecs.md#json) is `json`.
+
+Supported configuration operations are:
+
+- `hosts` - List of hosts to connect to.
+- `subject` - Subject for the to be published message.
+- `reply` - Optional string specifying the reply subject.
+- `headers` - Optional mapping specifying the headers for the message.
```

Comment:
Are there any restrictions on what header values can be?

Reply:
yeah both key and values need to be a String
`Optional mapping (String: Vec<String>) specifying the headers for the message.`
does this work?

Reply:
I would avoid the rust specific sones (i.e. Vec) perhaps something like `Headers are key-value pairs with the key being a string and the value being either a string or a list of strings`?